src/OrleansAzureUtils/Hosting/AzureClient.cs

```diff
@@ -123,12 +140,12 @@ private static void InitializeImpl_FromFile(FileInfo configFile)
             InitializeImpl_FromConfig(config);
         }
 
-        private static string GetDeploymentId()
+        internal static string GetDeploymentId()
         {
             return GrainClient.TestOnlyNoConnect ? "FakeDeploymentId" : serviceRuntimeWrapper.DeploymentId;
```

Comment:
I don't know. I don't even see it being used in tests. Probably a leftover from the past. But this PR isn't concerned with it.